lib/dbsearch.js

```diff
@@ -464,14 +466,16 @@ async function reIndexPids(pids, topic) {
 	await postsSave(postData);
 }
 
-async function renderAdmin(req, res) {
+async function renderAdmin (req, res) {
 	const results = await getGlobalAndPluginData();
 	results.plugin.progressData = await getProgress();
-	results.plugin.title = 'DB Search';
-	res.render('admin/plugins/dbsearch', results.plugin);
+	translator.translate('[[dbsearch:admin.title]]', function (translated) {
```

Comment:
Don't have to manually translate title I think, you can just set `results.plugin.title = '[[dbsearch:admin.title]]` and it should be translated.